kyuubi-server/src/main/antlr4/org/apache/kyuubi/sql/KyuubiSqlBaseLexer.g4

```diff
@@ -0,0 +1,105 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+lexer grammar KyuubiSqlBaseLexer;
```

Comment:
cc @ulysses-you, can you verify that this parser will work along with trino FE?

Reply:
it is not about trino for now, but we can support trino statement one by one by extending antlr later

Reply:
how can we tell from a meta operation and regular one for `select * from sys.tables` ？

Reply:
It won't affect other FE, that means other FE will pass though it. Only Trino session will match those special statements. I think user can not access the regular one if use Trino FE follows Trino's behavior.